### PR TITLE
[hi] localize docs/reference/glossary/cluster.md

### PR DESCRIPTION
**Pull request by kundan2707**
localize hi/docs/reference/glossary/cluster.md

### CODE REVIEW

**Comment by kundan2707** [comment]
@anubha-v-ardhan @divya-mohan0209  PATL

File: content/hi/docs/reference/glossary/cluster.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+tags:
+- fundamental
+- operation
+---
+वर्कर मशीनों का एक समूह, जिसे {{< glossary_tooltip text="नोड्स" term_id="node" >}} भी कहा जाता है,
+जो कंटेनरीकृत एप्लिकेशन्स चलाते हैं। हर क्लस्टर में कम से कम एक वर्कर नोड होता है।
+
+<!--more-->
+वर्कर मशीन {{< glossary_tooltip text="पॉड्स" term_id="pod" >}} को होस्ट करते है जो कि एप्लिकेशन वर्कलोड के घटक हैं।  
+{{< glossary_tooltip text="कंट्रोल प्लेन" term_id="control-plane" >}} क्लस्टर में वर्कर नोड्स और पॉड्स का प्रबंधन करता है। एक प्रोडक्शन  वातावरण में, कंट्रोल प्लेन आमतौर पर
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/cluster.md:19]:
```suggestion
{{< glossary_tooltip text="कंट्रोल प्लेन" term_id="control-plane" >}} क्लस्टर में वर्कर नोड्स और पॉड्स का प्रबंधन करता है। एक प्रोडक्शन वातावरण में, कंट्रोल प्लेन आमतौर पर
```

File: content/hi/docs/reference/glossary/cluster.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+कई कंप्यूटरों पर चलता है और एक क्लस्टर आमतौर पर कई नोड्स चलाता है, जो दोष-सहिष्णुता और उच्च उपलब्धता प्रदान करता है।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/cluster.md:20]: I don't know if high availability & failover should be translated the way you have done it here.

Comment (aj11anuj) [inline content/hi/docs/reference/glossary/cluster.md:20]:
```suggestion
कई कंप्यूटरों पर चलता है और एक क्लस्टर आमतौर पर कई नोड्स चलाता है, जो बिना किसी विफलता के उच्च उपलब्धता प्रदान करता है। 
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
Although it will change meaning of whole sentence a bit but will increase clarity a lot and i think that's fine here.

**Comment by kundan2707** [comment]
@divya-mohan0209 @aj11anuj  incorporated review comments 
PTAL.

File: content/hi/docs/reference/glossary/cluster.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+title: क्लस्टर
+id: cluster
+date: 2019-06-15
+full_link: 
+short_description: >
+  वर्कर मशीनों का एक समूह, जिसे नोड्स भी कहा जाता है, जो कंटेनरीकृत एप्लिकेशन्स चलाते हैं। हर क्लस्टर में कम से कम एक वर्कर नोड होता है।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/cluster.md:7]:
```suggestion
  वर्कर मशीनों का एक समूह, जिन्हें नोड्स भी कहा जाता है, जो कंटेनरीकृत एप्लिकेशन्स चलाते हैं। हर क्लस्टर में कम से कम एक वर्कर नोड होता है।
```

File: content/hi/docs/reference/glossary/cluster.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+aka: 
+tags:
+- fundamental
+- operation
+---
+वर्कर मशीनों का एक समूह, जिसे {{< glossary_tooltip text="नोड्स" term_id="node" >}} भी कहा जाता है,
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/cluster.md:14]:
```suggestion
वर्कर मशीनों का एक समूह, जिन्हें {{< glossary_tooltip text="नोड्स" term_id="node" >}} भी कहा जाता है,
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Since we're referring to the machines themselves and not just the group of machines, the associated pronouns also need to change in number.

**Comment by divya-mohan0209** [comment]
@kundan2707 : Please could you push an empty commit to retrigger the build?

**Comment by kundan2707** [comment]
> @kundan2707 : Please could you push an empty commit to retrigger the build?

Pushed empty commit.

**Review by aj11anuj — COMMENTED** [review]
/lgtm

**Pull request merged**